bindings/python/test_x86.py

```diff
@@ -142,7 +239,14 @@ def print_string_hex(comment, str):
         for r in regs_write:
             print(" %s" %(insn.reg_name(r)), end="")
         print("")
-
+        
+    if insn.eflags:
+        updated_flags = []
+        for i in range(0,46):
```

Comment:
to keep this simple, can we just change "46" to "64", because beyond 46, there is no flags anyway?